OpenRA.Mods.Common/Lint/CheckPlayers.cs

```diff
@@ -92,8 +92,12 @@ void Run(Action<string> emitError, Action<string> emitWarning, MapPlayers player
 				if (playerCount > spawnPoints.Length)
 					emitError($"The map allows {playerCount} possible players, but defines only {spawnPoints.Length} spawn points.");
 
-				if (spawnPoints.Distinct().Count() != spawnPoints.Length)
-					emitError("Duplicate spawn point locations detected.");
+				var spawns = spawnPoints.ToList();
```

Comment:
This doesn't seem very efficient. What about something like:
```csharp
foreach (var spawn in spawnPoints.GroupBy(x => x))
    if (spawn.Count() > 1)
         emitError($"Duplicate spawn point location detected at ({spawn.Key}).");
```

Reply:
It's probably of the same efficiency, but sure, it looks cleaner